ios/YMChatReactNative.m

```diff
@@ -98,4 +108,13 @@ - (void) onBotClose {
     }
 }
 
+- (UIColor *)getColorFromHexString:(NSString *)hexString {
+    unsigned rgbValue = 0;
+    NSScanner *scanner = [NSScanner scannerWithString:hexString];
+    [scanner setScanLocation:1]; // bypass '#' character
+    [scanner scanHexInt:&rgbValue];
+    return [UIColor colorWithRed:((rgbValue & 0xFF0000) >> 16)/255.0 green:((rgbValue & 0xFF00) >> 8)/255.0 blue:(rgbValue & 0xFF)/255.0 alpha:1.0];
```

Comment:
bypass '#' only if it exists in the first location

Reply:
ok